avocado/core/sysinfo.py

```diff
@@ -186,6 +186,8 @@ class Daemon(Command):
     :param logf: Basename of the file where output is logged (optional).
     :param compress_logf: Whether to compress the output of the command.
     """
+    def __init__(self, cmd):
```

Comment:
Now you're limiting the arguments and potentially breaking the API. The parent class can also take `logf` and `compress_log`, so you have to accept them here as well. By the way, both classes docstrings have a typo (`compress_logf`).